selection_tools/include/HistogramSet.hpp

```diff
@@ -12,20 +12,25 @@
 #include <string>
 
 #include <TH1D.h>
+#include <TH2D.h>
 
 #include "Particle.hpp"
 #include "Event.hpp"
 
 class HistogramSet {
 public:
-  HistogramSet(std::string prefix);
+  HistogramSet(std::string prefix, bool reduce_hists=false, bool alp=false);
```

Comment:
"alp" what? "reduce_hists" has quite obvious meaning, but "alp" doesn't correspond to an action ;)